packages/p2p-v2/src/components/AdvertiserName/__tests__/AdvertiserNameBadges.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { APIProvider } from '@deriv/api';
+import { render, screen } from '@testing-library/react';
+import AdvertiserNameBadges from '../AdvertiserNameBadges';
+
+const wrapper = ({ children }: { children: JSX.Element }) => <APIProvider>{children}</APIProvider>;
+let mockUseAdvertiserStats = {
+    data: {
+        isAddressVerified: false,
+        isIdentityVerified: false,
+        totalOrders: 10,
+    },
+    isLoading: true,
+};
+
+jest.mock('../../../hooks', () => ({
+    ...jest.requireActual('../../../hooks'),
+    useAdvertiserStats: jest.fn(() => mockUseAdvertiserStats),
+}));
+
+describe('AdvertiserNameBadges', () => {
+    it('should render loading', () => {
+        mockUseAdvertiserStats = {
+            data: {
```

Comment:
here as well, value looks same as above .. except for the totalOrders.. but i think value doesn't matter here since its not being tested